src/main/java/sg/ncl/MainController.java

```diff
@@ -182,6 +192,13 @@ public String tutorials() {
         return "tutorials";
     }
 
+    @RequestMapping("/maintainance")
+    public String maintainance() {
+        return "maintainance";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this method and declare a constant for this value. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS3400)